buildSrc/private/src/main/kotlin/androidx/build/AndroidXImplPlugin.kt

```diff
@@ -171,8 +171,8 @@ class AndroidXImplPlugin @Inject constructor(val componentFactory: SoftwareCompo
         project.pluginManager.withPlugin("org.jetbrains.kotlin.multiplatform") {
             project.extensions.configure(KotlinMultiplatformExtension::class.java) {
                 it.compilerOptions {
-                    languageVersion.set(KotlinVersion.KOTLIN_2_0)
-                    apiVersion.set(KotlinVersion.KOTLIN_2_0)
+                    languageVersion.set(KotlinVersion.KOTLIN_2_1)
+                    apiVersion.set(KotlinVersion.KOTLIN_2_1)
```

Comment:
This forces clients to use Kotlin 2.1. Please add to the release notes.

Also, If I am not mistaken, we already require it for non-jvm targets because of the ABI incompatibility?

Reply:
Adding @shishkin-pavel for review

Reply:
>  Please note that native and web klibs can be consumed only with Kotlin 2.1.0 or newer.

https://github.com/JetBrains/compose-multiplatform/releases/tag/v1.8.0

This is not about JVM. Now we might force the JVM libraries to use it as well. Please, check that, and if it is true, then add a Migration note

Reply:
Added

Reply:
Did you check that?